core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -378,7 +378,7 @@ <h2 class="activity-title protractor-test-exp-summary-tile-title">
                    style="background-color: <[collection.thumbnail_bg_color]>;">
               </div>
               <md-card class="oppia-activity-summary-tile top-card">
-                <a ng-href="<[getCollectionUrl(collection.id)]>">
+                <a ng-href="<[getCollectionUrl(collection.id)]>" style="color: black;">
```

Comment:
Can you see what happens if you use `text-decoration: none;` instead? That is the more canonical way to undo the effects of an "a" tag. However, if it does not work, we can stick to this.

Thanks!